src/main/scala/Joint.scala

```diff
@@ -92,6 +166,6 @@ case class Joint(normalVec: (Double, Double, Double), localOrigin: (Double, Doub
     * @return Distance relative to block origin (new local origin)
     */
   def updateJoint(blockOrigin: (Double, Double,Double)): Joint = {
```

Comment:
When we merge this to master, I'll make sure this function avoids recalculating the bounding sphere like we talked about.